docs/docsource/requirements.in

```diff
@@ -5,4 +5,6 @@ sphinxcontrib-plantuml == 0.14
 sphinxcontrib-svg2pdfconverter == 0.1.0
 sphinx-version-ref >= 0.0.1a1
 
--e git+https://github.com/scality/sphinx-tools.git@v1.1#egg=sphinx_scality
+-e git+https://github.com/scality/sphinx-tools.git@v3.0#egg=sphinx_scality
+# TODO: Add a meanful tag on the project
+-e git+https://github.com/scality/sphinx-copybutton.git@docs-ring-s3c#egg=sphinx_copybutton
```

Comment:
is this not on PyPI? It definitely should be... 😏

Reply:
@tcarmet: MeanINGful...
@miniscruff: Teachable moment here: up until now, I thought PyPi was a command that pointed to some mystic cloud. There is a repository as well?

Reply:
Yup you upload your packages to this index so that pip can download them.

Reply:
That takes much of the mystique out of that cloud.

Reply:
Oups, thanks @wabernatScality gonna fix that. Also worth noting that I have no plans in merging this as it is. (tried to open the PR as draft, but I failed :laughing: ) 

@miniscruff so `sphinx-copybutton`  is a real sphinx plugin and it's available on pypi. However I discovered that a fork has been made internally with some patches that I would even advice to make a PR upstream. That not being the case what's advisable to move forward here? My thinking is, to at least try to push some of the changes on the upstream repository. But there might be some of the patches that would not be "accepted" by the community, so we may have to "live" with the fork or find another way. I don't feel like uploading a fork of a project to pypi, but I'm curious to know if, even if it's a fork we should upload to pypi.

Reply:
Changes that reflect a global use case should be pushed up stream for sure, anything that will not can go into our fork. Uploading a fork to pypi is fine, just prefix it with scality and provide a link to the main base. Making sure we keep in line as much as possible. Also, making sure the license allows. Imo.